Dockerfile.ci

```diff
@@ -0,0 +1,14 @@
+FROM annotations_cli
+USER root
```

Comment:
we don't *have* to use `root`, but I'd like to have a different user than `www-data` so that files are not modifiable from the PHP processes

Reply:
possible solution: just use `www-data` now that we target `dev` and `ci` and worry about it later when `end2end`|`prod` images are produced

Reply:
Sounds like something to park. The future base eLife image would presumably come with an `elife` user.

Reply:
Checked out both the `root` road and the `www-data` road and it seems the first is the simplest as it's the default user of the Dockerfiles (while the third, `elife`, will be introduced later)